src/components/Popups/ReplyField/ItemList.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { SyntheticEvent } from 'react';
+import noop from 'lodash/noop';
+import DefaultItemRow from './DefaultItemRow';
+import { Collaborator } from '../../../@types';
+import './ItemList.scss';
+
+export type Props = {
+    activeItemIndex?: number;
+    filterString?: string;
+    itemRow?: JSX.Element;
+    items: Collaborator[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
+    setActiveItemIndex?: (index: number) => void;
+};
+
+const ItemList = ({
+    activeItemIndex = 0,
+    itemRow = <DefaultItemRow />,
+    items,
+    onSelect,
+    setActiveItemIndex = noop,
+    ...rest
+}: Props): JSX.Element => {
+    return (
+        <ul role="listbox" {...rest}>
+            {items.map((item, index) => {
+                if (!item) {
+                    return null;
+                }
+
+                return React.cloneElement(itemRow, {
```

Comment:
Can this be written using JSX? Maybe you could do something like https://github.com/box/box-annotations/blob/473305b2b479299f2cdeef31c09df8c51b18e1b9/src/components/AutoScroller/AutoScroller.tsx#L16 where `as` is renamed as `Element` and used to render

Reply:
> Can this be written using JSX? Maybe you could do something like
> 
> https://github.com/box/box-annotations/blob/473305b2b479299f2cdeef31c09df8c51b18e1b9/src/components/AutoScroller/AutoScroller.tsx#L16
> 
> where `as` is renamed as `Element` and used to render

Good idea. But for now I want to be consistent with BUIE, cause I have a plan to extract a basic Editor in BUIE, improve it, and import it in annotations in V2/V3.